modules/nf-core/foldseek/createdb/main.nf

```diff
@@ -0,0 +1,65 @@
+process FOLDSEEK_CREATEDB {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::foldseek=8.ef4e960"
```

Comment:
recommend moduleDir and environement.yml

Reply:
Totally forgot. Updated